src/Installer/Beta.htm

```diff
@@ -9,6 +9,9 @@
 		<h4>Release Notes</h4>
 		<ul>
 			<li>DEV_VERSION_NUMBER<ol>
+				<li>Chorus: Fix overflow error when receiving very large project data with resumable option.</li>
```

Comment:
Seems like you need to move "Fix overflow" into 2.2.15 and then add "Chorus: Upgrade Mercurial to version 3" as the current version.